devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -203,17 +217,34 @@ private getButtonMap() {[
 				"02" : 3,
 				"03" : 1,
 				"04" : 2
-		]
+		] ,
+        "ADUROLIGHT_CSC" : [
+                "01" : 1,
+                "02" : 2,
+                "03" : 3,
+                "04" : 4
+        ],
+        "Adurolight_NCC" : [
+                "01" : 1,
+                "02" : 2,
+                "03" : 3,
+                "04" : 4
+        ]
 ]}
 
 private getSupportedButtonValues() {
 	def values = ["pushed", "held"]
+    if ((device.getDataValue("model") == "Adurolight_NCC") || (device.getDataValue("model") == "ADUROLIGHT_CSC")) {
```

Comment:
you've used this same conditional in several places. It's probably best for readability to break it out into its own function. Perhaps `isAduroSmartRemote()`?